ros2.repos

```diff
@@ -69,9 +71,6 @@ repositories:
   launch:
     type: git
     url: https://github.com/ros2/launch.git
-    # Avoid an issue that breaks the MoveIt2 build: https://github.com/ros2/launch/issues/588
-    # TODO: Revert to 'master' when this issue is fixed
-    #version: master
```

Comment:
It's not possible with either the Ruby standard lib YAML parser (which is currently used) or PyYAML (a likely alternative for further work) to preserve comments in YAML documents. This comment has been "moved" to the human edited `spaceros.repos` file which contains overrides to be applied to the repos file generated by `rosinstall_generator` so it is preserved at the edit location even though it is not preserved here.